pipelines/dax/docker/build_imgserv.groovy

```diff
@@ -13,49 +13,34 @@ node('jenkins-master') {
 }
 
 notify.wrap {
-  def retries = 3
   def repo = 'webserv/imgserv'
   def tag = 'dax_latest'
 
-  def run = {
-    dir('imgserv/lsst-dm-ci') {
+  node('docker') {
```

Comment:
`utill.nodeWrap()` was added yesterday.  It would be a good idea to change this to `util.nodeWrap('docker')`.

Reply:
Changed.